frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -24,8 +25,13 @@ type ConsumedExtensions =
   | DashboardsTab
   | DashboardsCard
   | DashboardsOverviewHealthPrometheusSubsystem
-  | DashboardsOverviewQuery;
+  | DashboardsOverviewQuery
+  | ClusterServiceVersionAction;
 
+window.SERVER_FLAGS.prometheusBaseURL =
+  'http://prometheus-k8s-openshift-monitoring.apps.12aug.devcluster.openshift.com';
+window.SERVER_FLAGS.alertManagerBaseURL =
+  'https://alertmanager-main-openshift-monitoring.apps.12aug.devcluster.openshift.com';
```

Comment:
Remember to remove this before we're ready to merge

Reply:
ack